docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -155,21 +125,58 @@ imagePullSecrets:
 - name: "my-image-pull-secret"
 ```
 
-#### WebLogic domain management
+##### `enableClusterRoleBinding`
+Specifies whether the roles necessary for the operator to manage domains
+will be granted using a ClusterRoleBinding rather than using RoleBindings in each managed namespace.
 
-##### `domainNamespaces`
+Defaults to `false`.
+
+This option greatly simplifies managing namespaces when the selection is done using label selectors or
+regular expressions as the operator will already have privilege in any namespace.
 
-Specifies a list of WebLogic domain namespaces which the operator manages. The names must be lower case. You are responsible for creating these namespaces.
+Customers who deploy the operator in Kubernetes clusters that run unrelated workloads will likely
+not want to use enable this option.
 
-This property is required.
+If `enableClusterRoleBinding` is `false` and you select namespaces that the operator will
+manage using label selectors or a regular expression, then the Helm release will only include
+RoleBindings in each namespace that match at the time the Helm release is created. If you later
+create namespaces that the operator should manage, the new namespaces will not yet have the necessary
+RoleBinding.
+
+You can correct this by upgrading the Helm release and reusing values:
+```
+$ helm upgrade \
+  --reuse-values \
+  weblogic-operator \
+  kubernetes/charts/weblogic-operator
+```
+
+#### WebLogic domain management
+
+##### `domainNamespaceSelectionStrategy`
+Specifies how the operator will select the set of namespaces that it will manage. 
+Legal values are: `List`, `LabelSelector`, `RegExp`, and `Dedicated`. If set to `List`,
+then the operator will manage the set of namespaces listed by the `domainNamespaces` value.
+If set to `LabelSelector`, then the operator will manage the set of namespaces discovered by a list
+of namespaces using the value specified by `domainNamespaceLabelSelector` as a label selector.
+If set to `RegExp`, then the operator will manage the set of namespaces discovered by a list
+of namespaces using the value specified by `domainNamespaceRegExp` as a regular expression matched
+against the namespace names.
+Finally, if set to `Dedicated`, then operator will manage WebLogic Domains only in the same namespace
+which the operator itself is deployed, which is the namespace of the Helm release.
```

Comment:
Do we defaults `domainNamespaceSelectionStrategy` to ``List``?

Reply:
Yes, `domainNamespaceSelectionStrategy` defaults to `List`. I'll mention that here.

Reply:
I've updated the documentation to describe the default.